image.yaml

```diff
@@ -148,7 +148,7 @@ packages:
           - python-requests
           - python-enum34
           - PyYAML
-
+          - openssl
```

Comment:
Please remove this and add it to the module that requires it.  Modules should be self contained and not require additional configuration on the part of consumers.

Reply:
Thanks, Rob. Looking.